tests/integration/rerender-test.js

```diff
@@ -0,0 +1,147 @@
+import { setComponentTemplate } from '@ember/component';
+import templateOnly from '@ember/component/template-only';
+import { precompileTemplate } from '@ember/template-compilation';
+import {
+  find,
+  getSettledState,
+  render,
+  rerender,
+  setupContext,
+  setupRenderingContext,
+  teardownContext,
+  waitFor,
+} from '@ember/test-helpers';
+import { buildWaiter } from '@ember/test-waiters';
+import GlimmerComponent from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { hbs } from 'ember-cli-htmlbars';
+import { module, test } from 'qunit';
+
+module('rerender real-world', function (hooks) {
+  hooks.beforeEach(async function () {
+    await setupContext(this);
+    await setupRenderingContext(this);
+  });
+  hooks.afterEach(async function () {
+    await teardownContext(this);
+  });
```

Comment:
🤔  do we also need to call `await teardownRenderingContext(this)`? I _think_ we do, but maybe I made that obsolete at some point?

Reply:
nvm, looks like that was removed in v2.0.0 as one of the breakages 🎉